packages/core/src/App/Containers/Modals/tnc-status-update-modal/tnc-status-update-modal.tsx

```diff
@@ -27,12 +28,12 @@ const TncStatusUpdateModal = observer(() => {
         >
             <div className='tnc-status-update-modal'>
                 <Text size={isDesktop ? 's' : 'xs'} weight='bold'>
-                    <Localize i18n_default_text="Updated T&C's" />
+                    <Localize i18n_default_text='Accept updated terms and conditions' />
                 </Text>
                 <div className='tnc-status-update-modal__text-container'>
                     <Text size={isDesktop ? 'xs' : 'xxs'}>
                         <Localize
-                            i18n_default_text='Please review our updated <0>terms and conditions</0>.'
+                            i18n_default_text='To continue trading, review and accept our updated <0>terms and conditions</0>. Not accepting will lead to restricted access to your account. Need help? Contact us via <0>live chat</0>.'
                             components={[
                                 <StaticUrl
                                     key={0}
```

Comment:
@suisin-deriv  Shouldn't the live chat link open up the live chat instead, i.e:
```suggestion
                            i18n_default_text='To continue trading, review and accept our updated <0>terms and conditions</0>. Not accepting will lead to restricted access to your account. Need help? Contact us via <1>live chat</1>.'
                            components={[
                                <StaticUrl
                                    key={0}
                                    className='link'
                                    href='terms-and-conditions'
                                    is_eu_url={!is_cr_account}
                                />,
                                <span
                                key={1}
                                className='link link--orange'
                                onClick={() => window.LiveChatWidget?.call('maximize')}
                            />
                            ]}
```

Reply:
yeap missed this out 😓 Updated the code 🙏